oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/web/server/DefaultServerOAuth2AuthorizationRequestResolver.java

```diff
@@ -52,6 +53,7 @@
  * used to resolve the {@link ClientRegistration} and create the {@link OAuth2AuthorizationRequest}.
  *
  * @author Rob Winch
+ * @author Mark Heckler
  * @since 5.1
  */
 public class DefaultServerOAuth2AuthorizationRequestResolver
```

Comment:
Please apply same changes as requested in `DefaultOAuth2AuthorizationRequestResolver`